mep/pages/templates/pages/snippets/citation.html

```diff
@@ -1,6 +1,22 @@
 <div class="citation">
     <h2>Cite this document</h2>
     <p>
+        <span class="authors">
+        {% spaceless %}
+        {% for person in page.authors %}
+        {% with author=person.value %}
+            {% if forloop.first %}
+            <span>{{ author.last_name }}, {{ author.first_name }}</span>
+            {% elif forloop.last %}
+            <span>, and {{ author.name }}</span>
```

Comment:
This looks good, but one more case to handle: no comma here if there are only two authors